std_msgs/Header header
float32 position
float32 velocity
float32 effort
float32 motor_current
